src/book/03-guides/03-hardware/04-DARwIn-OP2-guide.mdx

```diff
@@ -0,0 +1,184 @@
+---
+section: Guides
+chapter: Hardware
+title: DARwIn OP2 Robot Restoration and Calibration Guide
+description: How to use the provided instructions for setting up, restoring, and calibrating the DARwIn OP2 robot system.
+slug: /guides/hardware/DARwIn-OP2-guide
```

Comment:
I think all our page names and slugs are in lower case, it would be good to continue that for consistency.